README.md

```diff
@@ -37,8 +37,8 @@ This will create a directory with configuration files for your node and install
 
 There are several add-on services available to extend the functionality of Bitcore:
 
-- [Insight API](https://github.com/bitpay/insight-api/tree/v0.3.0)
-- [Insight UI](https://github.com/bitpay/insight/tree/v0.3.0)
+- [Insight API](https://github.com/bitpay/insight-api)
+- [Insight UI](https://github.com/bitpay/insight)
```

Comment:
@braydonf 

Since "insight-ui" is the actual name of the repo, we should probably use that in the url here.

"https://github.com/bitpay/insight" does redirect to "https://github.com/bitpay/insight-ui", but it is probably best to use the non-redirecting url unless you have some special reason to do otherwise.